plugin/grepper.vim

```diff
@@ -936,6 +936,15 @@ function! s:run(flags)
           \ 'on_stderr': function('s:on_stdout_nvim'),
           \ 'on_exit':   function('s:on_exit'),
           \ }
+    if has('nvim-0.5.1')
+      " Starting with version 13, ripgrep always stats stdin and if it's not a
+      " TTY it uses it to read data. Unfortunately, Neovim always attaches a
+      " pipe to stdin by default and that leads to ripgrep reading nothing...
+      " (see https://github.com/neovim/neovim/pull/14812 for more info)
```

Comment:
```suggestion
      " (see https://github.com/mhinz/vim-grepper/issues/244 for more info)
```
The PR is referenced twice - maybe you've meant the issue here?  Or the one in ripgrep?

Reply:
Thanks for catching this, just updated the PR!